### DISCUSSION TITLE
Expose tailwindcss colors as standonle package

### DISCUSSION THREAD

**Discussion by joseDaKing** [tailwindlabs/tailwindcss#8134] [Ideas]
Expose tailwindcss colors as standonle package so it can be used in none tailwindcss projects

**Comment by dance2die** [comment]
You can get list of all Tailwind CSS colors from `tailwindcss/colors`.

e.g.
```
const colors = require("tailwindcss/colors");

// Get `pink 50`,
colors.pink[50]
```
